recipe/meta.yaml

```diff
@@ -1,4 +1,4 @@
-{% set version = "2019.10.11" %}
+{% set version = "2019.10.22" %}
```

Comment:
Perhaps we can hold off on the version number bump in this PR. Was planning on bumping the version in PR ( https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/309 ).

Reply:
ok